data-dashboard/server/trpc/routers/charts.ts

```diff
@@ -103,6 +110,21 @@ const chartRouter = router({
         throw new TRPCError({ code: 'INTERNAL_SERVER_ERROR', message: error.message });
       }
     }),
+  getChartData: privateProcedure
+    .input(GetChartDataSchema)
+    .query(async ({ ctx, input }) => {
+      const user = ctx.user();
+      const { chartId } = input;
+      await ctx.chartSerializer.buildChart('Deudas2.xlsx');
```

Comment:
I think we should remove this before mergin, or could crash

Reply:
I removed this query altogether. What do you think?